### PR TITLE
feat(css): Update css2 syntax for some pseudo-elements

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

the legacy info is helpless and useless, and does not fit the css syntax schema

### CODE REVIEW

File: css/selectors.json [hunk 1]
```diff
@@ -875,7 +875,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-webkit-slider-thumb"
   },
   "::after": {
-    "syntax": "/* CSS3 syntax */\n::after\n\n/* CSS2 syntax */\n:after",
+    "syntax": "::after",
```
Comment (skyclouds2001) [inline css/selectors.json:878]:
```suggestion
    "syntax": "::after | :after",
```

File: css/selectors.json [hunk 2]
```diff
@@ -893,7 +893,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::backdrop"
   },
   "::before": {
-    "syntax": "/* CSS3 syntax */\n::before\n\n/* CSS2 syntax */\n:before",
+    "syntax": "::before",
```
Comment (skyclouds2001) [inline css/selectors.json:896]:
```suggestion
    "syntax": "::before | :before",
```

File: css/selectors.json [hunk 3]
```diff
@@ -919,7 +919,7 @@
     "status": "standard"
   },
   "::first-letter": {
-    "syntax": "/* CSS3 syntax */\n::first-letter\n\n/* CSS2 syntax */\n:first-letter",
+    "syntax": "::first-letter",
```
Comment (skyclouds2001) [inline css/selectors.json:922]:
```suggestion
    "syntax": "::first-letter | :first-letter",
```

File: css/selectors.json [hunk 4]
```diff
@@ -928,7 +928,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::first-letter"
   },
   "::first-line": {
-    "syntax": "/* CSS3 syntax */\n::first-line\n\n/* CSS2 syntax */\n:first-line",
+    "syntax": "::first-line",
```
Comment (skyclouds2001) [inline css/selectors.json:931]:
```suggestion
    "syntax": "::first-line | :first-line",
```

File: css/selectors.json [hunk 5]
```diff
@@ -875,7 +875,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-webkit-slider-thumb"
   },
   "::after": {
-    "syntax": "/* CSS3 syntax */\n::after\n\n/* CSS2 syntax */\n:after",
+    "syntax": "::after | :after",
```
Comment (bsmth) [inline css/selectors.json:878]: Shouldn't we be targeting latest spec and remove the others?

```suggestion
    "syntax": "::after",
```

Comment (bsmth) [reply css/selectors.json:878]: Same comment for the ones below

File: css/selectors.json [hunk 6]
```diff
@@ -893,7 +893,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::backdrop"
   },
   "::before": {
-    "syntax": "/* CSS3 syntax */\n::before\n\n/* CSS2 syntax */\n:before",
+    "syntax": "::before | :before",
```
Comment (skyclouds2001) [inline css/selectors.json:896]:
```suggestion
    "syntax": "::before",
```

File: css/selectors.json [hunk 7]
```diff
@@ -919,7 +919,7 @@
     "status": "standard"
   },
   "::first-letter": {
-    "syntax": "/* CSS3 syntax */\n::first-letter\n\n/* CSS2 syntax */\n:first-letter",
+    "syntax": "::first-letter | :first-letter",
```
Comment (skyclouds2001) [inline css/selectors.json:922]:
```suggestion
    "syntax": "::first-letter",
```

File: css/selectors.json [hunk 8]
```diff
@@ -928,7 +928,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::first-letter"
   },
   "::first-line": {
-    "syntax": "/* CSS3 syntax */\n::first-line\n\n/* CSS2 syntax */\n:first-line",
+    "syntax": "::first-line | :first-line",
```
Comment (skyclouds2001) [inline css/selectors.json:931]:
```suggestion
    "syntax": "::first-line",
```

**Review by bsmth — APPROVED** [review]
ty!

**Pull request merged**